lib/gds_api/test_helpers/email_alert_api.rb

```diff
@@ -270,7 +270,7 @@ def get_subscription_response(
             "id" => id,
             "frequency" => frequency,
             "source" => "user_signed_up",
-            "ended_at" => ended ? Time.now.rfc3339 : nil,
+            "ended_at" => ended ? Time.now.to_datetime.rfc3339 : nil,
```

Comment:
Why has this changed? Can you move it to a separate commit and add an explanation?

Reply:
Done.